src/photos.cs

```diff
@@ -107,26 +113,32 @@ interface PHAsset {
 		[Export ("fetchKeyAssetsInAssetCollection:options:")]
 		PHFetchResult FetchKeyAssets (PHAssetCollection assetCollection, [NullAllowed] PHFetchOptions options);
 
+		[Mac (10,15, onlyOn64: true)]
 		[Static]
 		[Export ("fetchAssetsWithBurstIdentifier:options:")]
 		PHFetchResult FetchAssets (string burstIdentifier, [NullAllowed] PHFetchOptions options);
 
+		[Mac (10,15, onlyOn64: true)]
 		[Static]
 		[Export ("fetchAssetsWithOptions:")]
 		PHFetchResult FetchAssets ([NullAllowed] PHFetchOptions options);
 
-		[Deprecated (PlatformName.TvOS, 11,0)]
-		[Deprecated (PlatformName.iOS, 11,0)]
+		[Introduced (PlatformName.TvOS, 8,0, message: "Will be removed in a future release")]
+		[Deprecated (PlatformName.TvOS, 11,0, message: "Will be removed in a future release")]
+		[Introduced (PlatformName.iOS, 8,0, message: "Will be removed in a future release")]
+		[Deprecated (PlatformName.iOS, 11,0, message: "Will be removed in a future release")]
```

Comment:
Is there an API to be used instead? The message does not give much info, we should point to a new method to use.

Reply:
Thanks for catching these, I followed Sebastian's advice about removing the [Introduced] attribute and removing the messages that don't give us much info in the Enums.cs file but not this one. If the documentation does not give a good method to use instead, would I just clear the entire message for these as well? In this case, there are other functions that can fetch assets using different types of parameters. Should I say something along the lines of "Use other fetchasset methods"?